torch/csrc/jit/mobile/import.cpp

```diff
@@ -444,16 +447,17 @@ mobile::Module BytecodeDeserializer::deserialize(
   //
   auto bvals = readArchive("bytecode", mcu).toTuple()->elements();
 
-  c10::optional<std::vector<IValue>> debug_info_bvals;
-  if (reader_->hasRecord("mobile_debug.pkl")) {
-    debug_info_bvals = readArchive("mobile_debug", mcu).toTuple()->elements();
+  c10::optional<std::vector<IValue>> debug_handles;
+  if (reader_->hasRecord("mobile_debug_handles.pkl")) {
+    debug_handles =
+        readArchive("mobile_debug_handles", mcu).toTuple()->elements();
   }
-  parseMethods(bvals, debug_info_bvals, *mcu);
+  parseMethods(bvals, debug_handles, *mcu);
   auto meta_dict = readMobileMetadata(mcu);
   auto m = mobile::Module(readArchive("data", mcu).toObject(), meta_dict, mcu);
 #if defined(SYMBOLICATE_MOBILE_DEBUG_HANDLE)
   std::unique_ptr<MobileDebugTable> debug_table =
-      std::make_unique<MobileDebugTable>(reader_);
+      std::make_unique<MobileDebugTable>(reader_, compilation_unit_);
```

Comment:
This change is here because `compilation_unit_` is used to resolve type names. There was no need to resolve type names since types only appeared in ModuleInstanceInfo which we are now serializing as part of InlinedCallStack.